ci/dev/vscode.patch

```diff
@@ -248,19 +248,17 @@ index 1e16cde724..0000000000
 -target "12.4.0"
 -runtime "node"
 diff --git a/src/vs/base/common/network.ts b/src/vs/base/common/network.ts
-index e4546b2cf6..9df12239fb 100644
+index e4546b2cf6..ad2c544e89 100644
 --- a/src/vs/base/common/network.ts
 +++ b/src/vs/base/common/network.ts
-@@ -94,16 +94,18 @@ class RemoteAuthoritiesImpl {
+@@ -94,16 +94,17 @@ class RemoteAuthoritiesImpl {
  		if (host && host.indexOf(':') !== -1) {
  			host = `[${host}]`;
  		}
 -		const port = this._ports[authority];
 +		// const port = this._ports[authority];
```

Comment:
Should this be commented out?

Reply:
It's not used so it's commented out to prevent a build error about unused variables. Actually we could just remove the line instead of commenting it out.

As to why it's not used, VS Code normally sets the authority portion of the websocket to `host:port` but we change it to `window.location.host`. The problem is that they pull the host and port from the remote authority which is set based on the `host` header and that's not always accurate (reverse proxies, for example).

Reply:
Why wasn't it causing a build error before?